codaco/Network-Canvas
# Fix node form submit

Fixes #770. Overlay is now a container; its `ref` is obtainable from `getWrappedInstance()`.

### To test

Submit a node form; there should be no error. If the form is long (e.g. the "inputs" form in the dev protocol) it should again scroll to top.
